src/sentry/utils/env.py

```diff
@@ -1,3 +1,4 @@
+import logging
```

Comment:
This should be surrounded by TYPE_CHECKING

Reply:
logging is a `stdlib` module so it's not going to cause an import cycle -- trying to avoid the import here also doesn't save anything since the module using this module will have already imported `logging`